gtkm/auth/service_auth.py

```diff
@@ -1,13 +1,80 @@
-from fastapi import APIRouter
+from fastapi import APIRouter, Depends, HTTPException, Response, status
+from fastapi.encoders import jsonable_encoder
+from fastapi.responses import RedirectResponse, JSONResponse
+from sqlalchemy.orm import Session
+import requests
+import uuid
 
+from .database.database import Base, engine, get_db
+from .database.user_schema import User
+from .database.database_operations import get_user, get_user_by_github_login, create_user
+from .sessions.session import backend, cookie, verifier
+from .sessions.session_data import SessionData
+
+
+Base.metadata.create_all(bind=engine)
 auth = APIRouter()
 
+CLIENT_ID = "<CLIENT_ID>"
+CLIENT_SECRET = "<CLIENT_SECRET>"
 
+
+# Endpoint for obtaining user data based on the user ID
 @auth.get("/auth/user/")
```

Comment:
Czy nie powinien być tutaj zdefiniowany typ zwracanego obiektu?

Tak jak np w:  
`@auth.get("/api/auth/github/authorized", response_model=User)`

Reply:
Zrobione, w `/authorized` usunąłem ten typ (zwracany był inny niż User), dodałem za to w `/auth/user/`